tests/queries/0_stateless/03197_fix_parse_mysql_iso_date.sql

```diff
@@ -0,0 +1 @@
+SELECT parseDateTime(formatDateTime(toDateTime('2024-06-20 12:00:00'), '%F %T', 'Europe/Paris'), '%F %T', 'UTC') AS x
```

Comment:
If I may wish for one thing: Please put this at the end of tests/queries/0_stateless/02668_parse_datetime.sql (which already contains a few tests for crashes), so we keep related tests together.

Reply:
AFAIK we prefer creating new tests so we have a better history of its failure

Reply:
But I'm fine either way